tools/topology/sof-tgl-sdw-max98373-rt5682.m4

```diff
@@ -0,0 +1,240 @@
+#
+# Topology for Tigerlake with sdw rt5682 + Max98373 amp + DMIC + 4 HDMI
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`alh.m4')
+include(`hda.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Tigerlake DSP configuration
+include(`platform/intel/tgl.m4')
+include(`platform/intel/dmic.m4')
+
+DEBUG_START
+
+#
+# Define the pipelines
+#
+# PCM0 <---> volume <----> playback (Headset - ALC5682)
+# PCM1 <---> volume <----> capture  (Headset - ALC5682)
+# PCM2 ----> smart_amp ----> ALH0x102 (Speaker -max98373)
+#              ^
+#              |
+#              |
+# PCM3 <---- demux  <----- ALH0x103 (Speaker -max98373)
+# PCM4 <---- volume <---- DMIC01
+# PCM5 <---- volume <---- DMIC16k
```

Comment:
Do we have WoV requirement for SDW platform also?

Reply:
Not now

Reply:
Okay, we can add it later.